Dockerfile

```diff
@@ -55,6 +57,8 @@ RUN wget -c https://dl.orbbec3d.com/dist/openni2/ROS2/OpenNI_SDK_ROS2_v1.0.2_202
     mv ros2_astra_camera src && \
     rm -rf OpenNI_SDK_ROS2*
 
+COPY ./ob_camera_node.cpp /ros2_ws/src/ros2_astra_camera/astra_camera/src/ob_camera_node.cpp
```

Comment:
maybe we could create a fork instead? In this case push the code from tar to our github repo

Reply:
The fork is not possible because the version in [the repository](https://github.com/orbbec/ros_astra_camera/tree/ros2-development) is not the latest as the version from https://orbbec3d.com/index/download.html. 

Maybe orbbec didn't want to publish that on GH.

Reply:
I know, I was wondering if we could push the code from the download section to the github repo